### ISSUE TITLE
Ubuntu OM installation broken?

### ISSUE THREAD

**Issue by max-privato** [OpenModelica/OpenModelica#12468]
### Steps to Reproduce

I tried to install nightly on Ubuntu Jammy, but got the error below.
![image](https://github.com/OpenModelica/OpenModelica/assets/37046832/20da3c48-c093-427e-84e1-9f4c988f11d7)

It seems like Ubuntu does not recognize a correct signature on the software and therefore refuses to install.

**Comment by adrpo** [comment]
Did you installed the key?
https://openmodelica.org/download/download-linux/

```
sudo apt-get update
sudo apt-get install ca-certificates curl gnupg
sudo curl -fsSL http://build.openmodelica.org/apt/openmodelica.asc | \
  sudo gpg --dearmor -o /usr/share/keyrings/openmodelica-keyring.gpg
```

**Comment by adrpo** [comment]
I tried this with the Jammy and the latest Ubuntu docker image, works fine.

**Comment by max-privato** [comment]
> Did you installed the key? https://openmodelica.org/download/download-linux/

I thought I did. Maybe I made a mistake somewhere. 
Today I reinstalled OM successfully.

But... I may be foolish these days: I could not install he nightly. Indeed I get always the previous release installation.

Here the screenshot of all the steps that I've just repeated:
![image](https://github.com/OpenModelica/OpenModelica/assets/37046832/50c7e661-d6e9-4efb-9e55-9e6322b4e307)
